uint8 max_linear_velocity
uint8 max_angular_velocity
float32 P
float32 I
float32 D
bool send_voronoi
bool send_path
float32 acceleration
float32 deceleration
bool send_smooth_path
